src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -149,7 +149,7 @@ const TeamMembersPageBlock: React.FC<TeamMembersPageBlockProps> = ({ team }) =>
 
   const NonEditingMembersView = () => (
     <Grid container>
-      <Grid item xs={11} lg="auto">
+      <Grid item xs={11} lg="auto" style={{ maxWidth: 'calc(100% - 48px)' }}>
```

Comment:
Good job on adding `maxWidth` style here to wrap the members list. 

In fact, I think we can use the full width of this Grid to list members list since this Grid has limited width size already, as it's in `PageBlock` component as a child element.

In other words, I believe we can use `fit-content` instead of `calc(100% - 48px)` to wrap the text and avoid hard-coding the size value.

Reply:
Alternatively, I think setting the max width as `100%` should also work, but let's ask it to Reid after we test out for `fit-content`.